apps/web-giddh/src/app/multi-currency-reports/balance-sheet/components/balance-sheet-grid/components/balance-sheet-grid-row/balance-sheet-report-grid-row.component.scss

```diff
@@ -0,0 +1,20 @@
+.pl-grid-row {
+    flex-wrap: nowrap;
+}
+.group {
+    width: 30%;
+    max-width: 30%;
+    white-space: nowrap;
+    min-width: 150px;
+}
+.account {
+    div {
+        -webkit-line-clamp: 1;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
use class text-ellipsis

this class contains
    text-overflow: ellipsis;
    overflow: hidden;
    white-space: nowrap;

Reply:
Done